test/utils.py

```diff
@@ -253,55 +296,22 @@ def get_empty_chw_tensor(self, *, stream_index: int) -> torch.Tensor:
 NASA_VIDEO = TestVideo(
     filename="nasa_13013.mp4",
     default_stream_index=3,
-    # This metadata is extracted manually.
-    #  for stream index 0:
-    #    $ ffprobe -v error -hide_banner -select_streams v:0 -show_frames -of json test/resources/nasa_13013.mp4 > out.json
-    #
-    #  for stream index 3:
-    #    $ ffprobe -v error -hide_banner -select_streams v:1 -show_frames -of json test/resources/nasa_13013.mp4 > out.json
-    #
-    # Note that we are using the absolute stream index in the file. But ffprobe uses a relative stream
-    # for that media type.
     stream_infos={
         0: TestVideoStreamInfo(width=320, height=180, num_color_channels=3),
         3: TestVideoStreamInfo(width=480, height=270, num_color_channels=3),
     },
-    frames={
-        0: {
-            0: TestFrameInfo(pts_seconds=0.0, duration_seconds=0.040000),
-            1: TestFrameInfo(pts_seconds=0.040000, duration_seconds=0.040000),
-            2: TestFrameInfo(pts_seconds=0.080000, duration_seconds=0.040000),
-            3: TestFrameInfo(pts_seconds=0.120000, duration_seconds=0.040000),
-            4: TestFrameInfo(pts_seconds=0.160000, duration_seconds=0.040000),
-            5: TestFrameInfo(pts_seconds=0.200000, duration_seconds=0.040000),
-            6: TestFrameInfo(pts_seconds=0.240000, duration_seconds=0.040000),
-            7: TestFrameInfo(pts_seconds=0.280000, duration_seconds=0.040000),
-            8: TestFrameInfo(pts_seconds=0.320000, duration_seconds=0.040000),
-            9: TestFrameInfo(pts_seconds=0.360000, duration_seconds=0.040000),
-            10: TestFrameInfo(pts_seconds=0.400000, duration_seconds=0.040000),
-        },
-        3: {
-            0: TestFrameInfo(pts_seconds=0.0, duration_seconds=0.033367),
-            1: TestFrameInfo(pts_seconds=0.033367, duration_seconds=0.033367),
-            2: TestFrameInfo(pts_seconds=0.066733, duration_seconds=0.033367),
-            3: TestFrameInfo(pts_seconds=0.100100, duration_seconds=0.033367),
-            4: TestFrameInfo(pts_seconds=0.133467, duration_seconds=0.033367),
-            5: TestFrameInfo(pts_seconds=0.166833, duration_seconds=0.033367),
-            6: TestFrameInfo(pts_seconds=0.200200, duration_seconds=0.033367),
-            7: TestFrameInfo(pts_seconds=0.233567, duration_seconds=0.033367),
-            8: TestFrameInfo(pts_seconds=0.266933, duration_seconds=0.033367),
-            9: TestFrameInfo(pts_seconds=0.300300, duration_seconds=0.033367),
-            10: TestFrameInfo(pts_seconds=0.333667, duration_seconds=0.033367),
-            25: TestFrameInfo(pts_seconds=0.8342, duration_seconds=0.033367),
-            35: TestFrameInfo(pts_seconds=1.1678, duration_seconds=0.033367),
-        },
-    },
+    frames={},  # Automatically loaded from json file
```

Comment:
We have to create an empty `frames` field here. This field cannot have a default within the `TestContainerFile` definition, because [defaults aren't allowed for base dataclasses](https://stackoverflow.com/questions/51575931/class-inheritance-in-python-3-7-dataclasses)